core/src/main/java/io/lionweb/lioncore/java/serialization/JsonSerialization.java

```diff
@@ -21,6 +21,16 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
+/**
+ * When deserializing a tree, we either extract an entire partitions (and perhaps all the referred
+ * partitions) or we will get references to nodes (for example, parents or ancestors) outside of the
+ * scope of the tree extracted. This policy specifies what we do with such references.
+ */
+enum UnknownNodePolicy {
```

Comment:
This policy is only about unknown _parents_, correct? Unknown annotations / children / reference targets are not affected. We should reflect that in the name.

Reply:
Right, I will do that